### DISCUSSION TITLE
How do i access store created by `svelte-query` on some other page?

### DISCUSSION THREAD

**Discussion by nooblyf** [TanStack/query#4824] [Q&A] [answered]
As mentioned in the docs `svelte-query` returns data in a svelte store, but i can't find how do i access that store on some other nested page or do i need to create my own store and sync it with `svelte-query`'s store?

**Comment by louis-young** [comment, marked as answer]
Hi 👋 

I haven't used the Svelte adapter but I'd imagine you can access the data that exists in the cache for a given query by creating/using a query with the same query key wherever you need it, or imperatively via the query client depending on what exactly you're trying to achieve. 

Have you tried either of these things?

**Reply by nooblyf** [reply]
Thanks, that works! and to avoid unnecessary re-fetching i've added `staleTime: Infinity` and invalidating query by key only when needed

**Reply by louis-young** [reply]
Awesome! No worries 🚀
